radar_parlamentar/importadores/chefes_executivos.py

```diff
@@ -51,13 +51,24 @@ def criar_chefe_executivo(self, nome, sigla_partido, ano_inicio, ano_fim, genero
         chefe = models.ChefeExecutivo(nome=nome, partido=partido, mandato_ano_inicio=ano_inicio,
                                       mandato_ano_fim=ano_fim, genero=genero)
 
-        chefe_existe = self.verificar_chefe_existe(chefe)
-        if not chefe_existe:
+        self.salvar_chefe_executivo(chefe)
+
+
+    def salvar_chefe_executivo(self, chefe):
+
+        chefe_existe, chefe_atual = self.verificar_chefe_existe(chefe) #chefe_atual é recebido para adicionarmos relacão com outra casa
+
+        #Adiciona novo chefe ou adiciona casa a um chefe ja existente ou ignora chefe ja existente de acordo com chefe_existe
+        if (chefe_existe == False):
             chefe.save()
             chefe.casas_legislativas.add(self.casa)
-            logger.info('Adicionando chefe %s' % nome)
-        else:
-            logger.warn('Chefe %s já existe' % nome)
+            logger.info('Adicionando chefe %s' % chefe.nome)
+        elif (chefe_existe == True):
+            if not self.verifica_casa_existe_no_chefe(chefe_atual):
+                chefe_atual.casas_legislativas.add(self.casa)
+                logger.info('Adicionando chefe %s em outra casa' % chefe_atual.nome)
+            else:
+                logger.warn('Chefe %s já existe' % chefe.nome)
 
 
     def verificar_chefe_existe(self, chefe):
```

Comment:
Humm... a assinatura deste método ficou confusa. Acho que a função poderia retornar só o chefe_executivo, e se esse cara for None, daí quer dizer q ele não existe no banco. Nesse caso, o nome do método deveria ser alterado para alguma coisa como "get_chefe_executivo_do_banco".